server/schema/volunteers.json

```diff
@@ -0,0 +1,7 @@
+{
+	"full_name": "{{chance.first()}} {{chance.last()}}",
+	"phone": "{{chance.phone()}}",
+	"email": "{{chance.email()}}",
+	"shirt_size": "{{util.sample(['X-Small', 'Small', 'Medium', 'Large', 'X-Large'])}}",
+	"availability": ""

```

Comment:
Should that be an array since the event will be over the span of 3 days?

Reply:
look at the typeform I made for the volunteer listing, I added the time slots we are looking for there and a list of days
we should have those as two separate fields